PoSSapi/src/Domain/Entities/Business.cs

```diff
@@ -0,0 +1,8 @@
+namespace PoSSapi.Domain.Entities;
+
+public class Business
+{
+    public Guid Id { get; set; }
+    public ICollection<BusinessLocation> Location { get; set; }
+    //Needs BusinessType enum but I am unsure what values it could have
```

Comment:
`ManagerId` should be here
![image](https://user-images.githubusercontent.com/73688133/208237451-288831cc-611d-4f08-904b-dc2278c78df3.png)

Reply:
Maybe business types: Restaurant, Service? Would be good enough